src/components/Media.js

```diff
@@ -573,10 +575,60 @@ export default class Video extends Component {
         >
           {this.renderChildren()}
         </video>
+      );
+    }
+    return (
+      <audio
+        className={classNames('video-react-video', this.props.className)}
+        id={videoId}
+        crossOrigin={crossOrigin}
+        ref={c => {
+          this.video = c;
+        }}
+        muted={muted}
+        preload={preload}
+        loop={loop}
+        playsInline={playsInline}
+        autoPlay={autoPlay}
+        poster={poster}
+        src={src}
+        onLoadStart={this.handleLoadStart}
+        onWaiting={this.handleWaiting}
+        onCanPlay={this.handleCanPlay}
+        onCanPlayThrough={this.handleCanPlayThrough}
+        onPlaying={this.handlePlaying}
+        onEnded={this.handleEnded}
+        onSeeking={this.handleSeeking}
+        onSeeked={this.handleSeeked}
+        onPlay={this.handlePlay}
+        onPause={this.handlePause}
+        onProgress={this.handleProgress}
+        onDurationChange={this.handleDurationChange}
+        onError={this.handleError}
+        onSuspend={this.handleSuspend}
+        onAbort={this.handleAbort}
+        onEmptied={this.handleEmptied}
+        onStalled={this.handleStalled}
+        onLoadedMetadata={this.handleLoadedMetaData}
+        onLoadedData={this.handleLoadedData}
+        onTimeUpdate={this.handleTimeUpdate}
+        onRateChange={this.handleRateChange}
+        onVolumeChange={this.handleVolumeChange}
+        tabIndex="-1"
```

Comment:
why -1? will keyboard users still be able to navigate to it?

Reply:
@jnhooper - That is... a very good question. I think that tabIndex came with video-react. It looks like you can't tab to the overall player itself and it jumps to the progress bar first.

Reply:
I also don't know the exact answer to this off-hand, but I'm glad it came up, because I suspect this is worth further investigation. Though I don't consider it blocking (and it sounds like John doesn't either).